services/ui-src/src/components/PortalTable.tsx

```diff
@@ -373,6 +373,12 @@ export default function PortalTable<V extends {} = {}>({
                     {row.cells.map((cell, index) => {
                       return (
                         <td
+                          className={
+                            headerGroups[0].headers[index].id ===
+                            "packageStatus"
+                              ? "status-bandage"
+                              : ""
+                          }
```

Comment:
I feel like a better way to do this would be thru the cell property of the columns definition passed to PortalTable rather than enforcing style using a particular column name from within the PortalTable component itself.